src/AccessibilityInsights.SharedUx/Properties/Resources.resx

```diff
@@ -1506,4 +1500,7 @@ We collect anonymized data to identify the top accessibility issues found by use
   <data name="hlHelpName" xml:space="preserve">
     <value>Help</value>
   </data>
+  <data name="InvalidConfiguration" xml:space="preserve">
+    <value>Invalid Configuration Format!</value>
```

Comment:
>Format [](start = 33, length = 6)

This isn't an informative message, since it doesn't tell the user what to do or expect next. Can we improve that?

Reply:
I think it should be enough, but I can ask Risa later about it